src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -35,9 +32,10 @@ public class JsonOperationContext : IDisposable
 
         private int _lastStreamSize = 4096;
 
-        public JsonOperationContext(UnmanagedBuffersPool pool)
+        public JsonOperationContext()
         {
-            Pool = pool;
+            _arenaAllocator = new ArenaMemoryAllocator();
+            _arenaAllocatorForLongLivedValues = new ArenaMemoryAllocator(16 * 1024);
```

Comment:
What you are missing here is that:
a) This is for more than just strings.
b) The context is pooled, we aren't calling dispose on it all that often